server/health_grpc.go

```diff
@@ -6,33 +6,34 @@ import (
 
 	"github.com/cybozu-go/log"
 	"github.com/cybozu-go/moco"
-	"github.com/cybozu-go/moco-agent/server/proto"
 	"github.com/cybozu-go/moco/accessor"
 	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/health"
+	healthpb "google.golang.org/grpc/health/grpc_health_v1"
 	"google.golang.org/grpc/status"
 )
 
 // NewHealthService creates a new HealthServiceServer
-func NewHealthService(agent *Agent) proto.HealthServiceServer {
+func NewHealthService(agent *Agent) healthpb.HealthServer {
 	return &healthService{
 		agent: agent,
 	}
 }
 
 type healthService struct {
-	proto.UnimplementedHealthServiceServer
+	health.Server
 	agent *Agent
 }
 
-func (s *healthService) Health(ctx context.Context, req *proto.HealthRequest) (*proto.HealthResponse, error) {
+func (s *healthService) Check(ctx context.Context, in *healthpb.HealthCheckRequest) (*healthpb.HealthCheckResponse, error) {
 	db, err := s.agent.acc.Get(fmt.Sprintf("%s:%d", s.agent.mysqlAdminHostname, s.agent.mysqlAdminPort), moco.MiscUser, s.agent.miscUserPassword)
 	if err != nil {
 		log.Error("failed to connect to database before health check", map[string]interface{}{
 			"hostname":  s.agent.mysqlAdminHostname,
 			"port":      s.agent.mysqlAdminPort,
 			log.FnError: err,
 		})
-		return nil, status.Errorf(codes.Internal, "failed to connect to database before health check: err=%v", err)
+		return &healthpb.HealthCheckResponse{Status: healthpb.HealthCheckResponse_SERVICE_UNKNOWN}, status.Errorf(codes.Internal, "failed to connect to database before health check: err=%v", err)
```

Comment:
Can we use the `SERVICE_UNKNOWN` in the `Check` method?

https://github.com/grpc/grpc/blob/master/doc/health-checking.md
>    SERVICE_UNKNOWN = 3;  // Used only by the Watch method.